### PR TITLE
Add fuzz input maintenance docs to the readme

### PR DESCRIPTION
**Pull request by dergoegge**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,34 @@
...
+
+Over time fuzz engines reduce inputs (produce a smaller input that yields the
+same coverage statistics). This causes our copora to accumulate larger
+non-reduced inputs.
+
+To avoid corpora bloat and potential CI timeouts, we usually prune/minimize our
+copora around the branch-off point using the
+[`delete_nonreduced_fuzz_inputs.sh`](https://raw.githubusercontent.com/bitcoin-core/bitcoin-maintainer-tools/main/delete_nonreduced_fuzz_inputs.sh)
+script (Recommended to run in a fresh VM, see documentation in the script).
+
```
[the remark this answers is not in the document]
Comment (dergoegge) [reply README.md:31]: By running twice you mean running it twice on the non-reduced corpus to try and see if the result is similar? Or running it again on the reduced corpus from the first run?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,39 @@
...
+* Code changes can lead to inputs loosing their coverage.
+
+To avoid corpora bloat, stale inputs and potential CI timeouts, we usually
+prune/minimize our copora around the branch-off point using the
+[`delete_nonreduced_fuzz_inputs.sh`](https://raw.githubusercontent.com/bitcoin-core/bitcoin-maintainer-tools/main/delete_nonreduced_fuzz_inputs.sh)
+script (Recommended to run in a fresh VM, see documentation in the script). The
+script is usually run twice to ensure that the results are "somewhat"
+reproducible.
+
+After pruning the corpora, the coverage should not have dropped significantly.
```
[the remark this answers is not in the document]
Comment (dergoegge) [reply README.md:35]: If the code for a target changed and that invalidated the inputs, then we might see a drop in coverage, no?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,40 @@
...
+
+### Pruning inputs 
+
+* Over time fuzz engines reduce inputs (produce a smaller input that yields the
+  same coverage statistics), which causes our copora to accumulate larger
+  non-reduced inputs. 
+* Code changes can lead to inputs losing their coverage.
+
+To avoid corpora bloat, stale inputs and potential CI timeouts, we usually
+prune/minimize our copora around the branch-off point using the
```
[the remark this answers is not in the document]
Comment (dergoegge) [reply README.md:29]: Will do once this is merged.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,34 @@
...
+### Contributing inputs
+
+*For documentation on how to fuzz Bitcoin Core please see
+[fuzzing.md](https://github.com/bitcoin/bitcoin/blob/master/doc/fuzzing.md).*
+
+If you want to contribute fuzz inputs, please "merge" the inputs before
+submitting a pull request. You can use the libFuzzer option `-merge=1`
+(recommended with `-use_value_profile=1`) or the `--m_dir` option of the fuzz
+runner:
+[`test_runner.py`](https://github.com/bitcoin/bitcoin/blob/master/test/fuzz/test_runner.py).
```
[the remark this answers is not in the document]
Comment (LadyS8214) [reply README.md:19]: LadyS8214

**Review by LadyS8214 — APPROVED** [review]

**Pull request merged**